arangodb-net-standard.Test/AqlFunctionApi/AqlFunctionApiClientTest.cs

```diff
@@ -225,5 +225,23 @@ public async Task PutChangeQueryTrackingConfigurationAsync_ShouldSucceed()
             Assert.False(putResponse.Error);
             Assert.Equal(HttpStatusCode.OK, putResponse.Code);
         }
+
+
+        [Fact]
+        public async Task GetQueryRulesAsync_ShouldSucceed()
+        {
+            if (_fixture.VersionMajor >= 3 && _fixture.VersionMinor >= 10)
```

Comment:
Should we implement this check using the `ServerVersion` check like [here](https://github.com/ArangoDB-Community/arangodb-net-standard/pull/386/files#diff-ed8d48f866d0bae6d3b75748f332abaac4deb764017aba52ed491c1ee3095e95)?

```c#
[Fact]
[Trait("ServerVersion", "3_10_PLUS")]
public async Task GetQueryRulesAsync_ShouldSucceed() { }
```

Or was this meant to be a new approach?

Both approaches have pros and cons. The Trait one is more visible, but a bit cumbersome as we need to update the test command in the config file for each version.